packages/react-components/react-textarea/src/components/Textarea/Textarea.types.ts

```diff
@@ -28,7 +28,7 @@ export type TextareaProps = Omit<
    *
    * @default outline
    */
-  appearance?: 'outline' | 'filled-darker' | 'filled-lighter';
+  appearance?: 'outline' | 'filled-darker' | 'filled-lighter' | 'filled-darker-shadow' | 'filled-lighter-shadow';
```

Comment:
I think this is a fine API change but I also think we should review it as part of a larger team sync since we'll be committing to it for the long haul.

Reply:
I was also thinking of the prop `shadow`, but it wouldn't make sense since only `filled-darker` and `filled-lighter` have shadows.

Reply:
Agreed, `shadow` prop is not the way to go. To be clear, I think what you have is where we'll end up but I just want to make sure we get lots of eyes on it first.